docs/install.md

```diff
@@ -1,51 +1,73 @@
+ * `Open Angel arena folder` is a folder name `oaa` inside of `addons folder`, it contains all the files which are present in this github repo 
+
+**`Dota2 addons folder` and `Open Angel arena folder` may not exist for the first time, they will be created after `Prepare Addon Directory` step**
+
+Final structure sholud look like `C:\(some path to steam library)\Steam\steamapps\common\dota 2 beta\addons\oaa\`
+
+### Navigating with command prompt
+Use command `cd` and the undergoing path to navigate,(`cd` works relative to your current path), for example
+```
+dota 2 beta\addons> cd oaa
+```
+will go from `Addons` folder to `Open Angel Arena` folder, and
+```
+dota 2 beta\addons\oaa> cd ../
 ```
-npm install -g gulp-cli dota2-addon-manager
+will navigate you back a step to `Addons` folder, and
 ```
-This will install both `gulp`, used for building npc files, and `d2am`, used to manage the addons directory.
+C:\somerandompath> D:
+```
+will change your current active drive to D:
 
-### Prepare Addon Directory
-Next find your `dota 2 beta` folder. You must change the directory of the node.js command prompt to this folder, this can be done with the `cd` command. It should look something like this...
+### Install Node Dependencies
+The global install for some reason works very confusingly, so i recommend using local repo for gulp.
+Now we navigate into `Dota2 installation folder` folder, and first install dependencies
 ```
-cd C:\Program Files (x86)\Steam\steamapps\common\dota 2 beta\
+npm install dota2-addon-manager
 ```
-**Note**: The directory path may be different depending on how you installed Steam. If Steam is installed on another drive you'll need to change drives using the letter then a `:` symbol. For example, `E:`.
 
-To ensure the addons directory is set up, run
+### Download and Deploy Project
+To ensure the addons directory is set up, navigate to `Dota2 Installation` folder and run
 ```
 d2am list
 ```
-leave the command prompt open, we'll come back to it. Join us on iscordDay, the URL ends in WNFBB4d. You found the easter egg!
-
-## Get The Addon
-Use the Github client to check out this addon into the addons directory. You can use the "Clone or Download" button right on the main page to open it using the desktop tool. Make sure it ends up in addons, the path should end in `dota 2 beta\addons\oaa\`.
-
-**You must link the addon before it will work**. You can do this from that same node.js window you had open by running
+This will create `Dota2 addons folder` folder and set it up!
+Now you have to open `GitHub Client` and using top left `+` button `clone` a repo `OAA` to your newly created `Dota2 addons folder`
+To become more familiar with `Git` and `GitHub` you can read a [guide for newcomers](/docs/github-for-noobs.md)
+Now we need to link this project to `Dota 2 Tools` with command
 ```
 d2am link
 ```
-You will see output about creating links for `oaa` game and content.
```

Comment:
please do not remove this note, many people needed it to know the command succeeded. Also this has nothing to do with gulp.